README.md

```diff
@@ -0,0 +1,29 @@
+# OpenTicket
+
+Let's build an open source, low commission (Roundtrip commission `< Max(0.25, 0.025 * TICKET_PRICE)`) event ticket distribution and marketplace.
+
+## Ticket Distribution
+
+Connect with entertainers and venues to generate, (re-)sell, and distribute tickets.
+
+**Basic requirements**
+
+- Generate tickets for set of seat identifiers.
+  - Seat mapping.
+  - General admission.
+  - VIP/Add-ons.
```

Comment:
yikes

Reply:
yikes what asshole

Reply:
Don't think this HAS to be included is what we meant for MVP.